packages/workbox-background-sync/src/lib/request-manager.js

```diff
@@ -18,8 +19,15 @@ class RequestManager {
    *
    * @private
    */
-  constructor({callbacks, queue}) {
-    this._globalCallbacks = callbacks || {};
+  constructor({callbacks, queue} = {}) {
+    callbacks = callbacks || {};
+
+    // Rename deprecated callbacks.
+    const base = 'workbox-background-sync.RequestManager.callbacks';
+    deprecate(callbacks, base, 'onResponse', 'replayDidSucceed');
```

Comment:
Yeah, this was an oversight and noticed it when updating the GA stuff. It should be fixed now.